src/timers/timers.rs

```diff
@@ -2,14 +2,13 @@ use std::collections::HashMap;
 use std::ops::{Deref, DerefMut};
 
 cfg_if::cfg_if! {
-    if #[cfg(feature="wasm")] {
+    if #[cfg(target_family = "wasm")] {
         use web_time::{Duration, Instant};
     }
     else {
         use std::time::{Duration, Instant};
     }
 }
```

Comment:
@goulart-paul Is there a reason to use cfg-if here ? (Instead of just adding the `#cfg` attribute directly ? 

https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg-attribute

Reply:
I think either way works.    Maybe the current way is slightly preferable since it presents the cases as mutually exclusive options?    

We already depend on cfg-if anyway since it is used when configuring basic types with/without BLAS support, and there cfg-if is quite useful since the `else` blocks can have (marginally) more substance.